lib/liquid/combinators/tags/custom_block.ex

```diff
@@ -46,6 +40,43 @@ defmodule Liquid.Combinators.Tags.CustomBlock do
     end
   end
 
+  @doc """
+  Check if the name parsed is not a `Liquid` tag.
```

Comment:
`name parsed is a liquid` instead of `name parsed is not a liquid`

Reply:
done